src/policy_metadata.rs

```diff
@@ -129,19 +129,24 @@ pub struct Metadata {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub annotations: Option<HashMap<String, String>>,
     pub mutating: bool,
+    #[serde(default = "_default_true")]
+    pub background_audit: bool,
     #[serde(default)]
     pub context_aware: bool,
     #[serde(default)]
     pub execution_mode: PolicyExecutionMode,
 }
 
+const fn _default_true() -> bool { true }
```

Comment:
This could be done with [serde_aux bool_true](https://docs.rs/serde-aux/latest/serde_aux/field_attributes/fn.bool_true.html), but feels like overkill.